docs/pages/enroll-resources/application-access/application-access.mdx

```diff
@@ -1,30 +1,106 @@
+template: doc-page
 tags:
  - zero-trust
  - infrastructure-identity
 ---
 
-The guides in this section explain how to protect applications with
-Teleport by deploying the Teleport Application Service.
+import DocHero from "@site/src/components/Pages/Landing/DocHero";
+import UseCasesList from "@site/src/components/Pages/Landing/UseCasesList";
+import Resources from "@site/src/components/Pages/Homepage/Resources";
 
-It is also possible to enroll applications in your cluster by deploying the
-Teleport SAML identity provider and configuring the applications as service
-providers. For more information, read [Configure Teleport as an identity
-provider](../../identity-governance/idps/idps.mdx).
+import userCheckSvg from "@site/src/components/Icon/svg/user-check.svg";
+import slidersHorizontalSvg from "@site/src/components/Icon/svg/sliders-horizontal.svg";
+import graphSvg from "@site/src/components/Icon/svg/graph.svg";
+import questionSvg from "@site/src/components/Icon/svg/question2.svg";
 
-## Getting started
-- [Introduction to Enrolling Applications](./introduction.mdx): How to set up Teleport to protect applications and cloud provider APIs
-- [Protect a Web Application with Teleport](./getting-started.mdx): Provides instructions to set up the Teleport Application Service and enable secure access to a web application.
+<DocHero
+  title="Enroll your first application"
+  youtubeVideoId="cvW4b96aPL0"
+  ctaLinks={[
+    {
+      title: "Get started with application access",
+      href: "./getting-started/",
+      variant: "secondary",
+      arrow: true,
+    },
+  ]}
+  links={[
+    {
+      title: "Role-based access control",
+      description: "Set up granular access policies for applications connected to Teleport.",
+      href: "./controls/",
+      icon: userCheckSvg,
+    }, 
+    {
+      title: "Configure Teleport as an IdP",
+      description: "Securely log into internal and external apps with Teleport as an identity provider.",
+      href: "./cloud-apis/",
+      icon: slidersHorizontalSvg,
+    },
+    {
+      title: "VNet: Build without VPNs",
+      description: "Secure app and SSH access with no VPNs or proxies",
+      href: "./guides/vnet/",
+      icon: graphSvg,
+    }, 
+    {
+      title: "Troubleshooting",
+      description: "Common application access problems and their solutions.",
+      href: "./troubleshooting-apps/",
+      icon: questionSvg,
+    },
+  ]}
+>
+Teleport provides secure access to applications and cloud provider APIs with RBAC, audit logging, and just-in-time access. 
 
-## Guides
-- [Guides (section)](./guides/): Guides for configuring Teleport application access.
-- [Securing Access to Cloud APIs (section)](./cloud-apis/): How to use Teleport to achieve secure access while managing your cloud-based infrastructure.
-- [JWT Authentication (section)](./jwt/): Guides for using Teleport application access JWT authentication.
```

Comment:
Seems JWT auth is also dropped from the new design. Should we add it to the "Application access guides" section?